### PR TITLE
Issue:3900446:Fix existing CI for PDR plugin

### PR DESCRIPTION
**Pull request by vg12345**

### CODE REVIEW

**Comment by vg12345** [comment]
bot:retest

**Review by TalLerner — APPROVED** [review]

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/dummy_trigger.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# Trigger 1
```
Comment (TalLerner) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/dummy_trigger.py:1]: why do we need this file?

**Comment by TalLerner** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Comment by boazhaim** [comment]
bot:retest

**Comment by boazhaim** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Comment by boazhaim** [comment]
bot:retest

**Comment by boazhaim** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Comment by vg12345** [comment]
bot:retest

**Pull request closed without merging**